scanpy/plotting/_stacked_violin.py

```diff
@@ -544,36 +551,43 @@ def stacked_violin(
         Alternatively, a single color name or hex value can be passed,
         e.g. `'red'` or `'#cc33ff'`.
     {show_save_ax}
-    **kwds
+    kwds
         Are passed to :func:`~seaborn.violinplot`.
 
     Returns
     -------
-    If `return_fig` is `True`, returns a :class:`StackedViolin` object,
+    If `return_fig` is `True`, returns a :class:`~scanpy.pl.StackedViolin` object,
     else if `show` is false, return axes dict
 
+    See also
+    --------
+    :class:`~scanpy.pl.StackedViolin`
+
+    :func:`~scanpy.pl.rank_genes_groups_stacked_violin` to plot marker genes identified
+        using the :func:`~scanpy.tl.rank_genes_groups` function.
+
     Examples
     -------
+
     >>> import scanpy as sc
     >>> adata = sc.datasets.pbmc68k_reduced()
     >>> markers = ['C1QA', 'PSAP', 'CD79A', 'CD79B', 'CST3', 'LYZ']
     >>> sc.pl.stacked_violin(adata, markers, groupby='bulk_labels', dendrogram=True)
 
     Using var_names as dict:
+
     >>> markers = {{'T-cell': 'CD3D', 'B-cell': 'CD79A', 'myeloid': 'CST3'}}
     >>> sc.pl.stacked_violin(adata, markers, groupby='bulk_labels', dendrogram=True)
 
     Get StackedViolin object for fine tuning
+
     >>> vp = sc.pl.stacked_violin(adata, markers, 'bulk_labels', return_fig=True)
     >>> vp.add_totals().style(ylim=(0,5)).show()
 
-    The axes used can be obtained using the get_axes() method
+    The axes used can be obtained using the get_axes() method:
+
     >>> axes_dict = vp.get_axes()
 
-    See also
-    --------
-    rank_genes_groups_stacked_violin: to plot marker genes identified using
-    the :func:`~scanpy.tl.rank_genes_groups` function.
```

Comment:
Now that we know that “see also” sections are definition lists: should this still be deleted?

Reply:
It was moved before the examples. But it seems it have an unneeded space.